experiments/ClimaEarth/cli_options.jl

```diff
@@ -2,107 +2,138 @@ import ArgParse
-        "--config_file"
-        help = "A yaml file used to set the configuration of the coupled model"
-        arg_type = String
-        "--job_id"
-        help = "A unique identifier for this run"
-        arg_type = String
-        default = nothing
-        "--print_config_dict"
-        help = "Boolean flag indicating whether to print the final configuration dictionary"
+        # Diagnostics information
+        "--use_coupler_diagnostics"
+        help = "Boolean flag indicating whether to compute and output coupler diagnostics [`true` (default), `false`]"
         arg_type = Bool
         default = true
-        "--FLOAT_TYPE"
-        help = "Floating point precision  [`Float64` (default), `Float32`]"
-        arg_type = String
-        default = "Float64"
-        "--coupler_toml_file"
-        help = "A toml file used to overwrite the model parameters. If nothing is specified, the default parameters are used."
+        # Physical simulation information
         "--evolving_ocean"
-        help = "Boolean flag indicating whether to use a dynamic slab ocean model or constant surface temperatures"
+        help = "Boolean flag indicating whether to use a dynamic slab ocean model, as opposed to constant surface temperatures [`true` (default), `false`]"
         arg_type = Bool
         default = true
-        "--device"
-        help = "Device type to use [`auto` (default) `CPUSingleThreaded`, `CPUMultiThreaded`, `CUDADevice`]"
+        "--mono_surface"
+        help = "Boolean flag indicating whether (1st order) monotone and conservative remapping is applied. [`false` (default), `true`]"
+        arg_type = Bool
+        default = false
+        "--turb_flux_partition"
+        help = "Method to partition turbulent fluxes. [`PartitionedStateFluxes`, `CombinedStateFluxesMOST` (default)]"
         arg_type = String
-        default = "auto"
-        "--use_coupler_diagnostics"
-        help = "Boolean flag indicating whether to compute and output coupler diagnostics [`true` (default), `false`]"
+        default = "CombinedStateFluxesMOST"
+        # Conservation information
+        "--energy_check"
+        help = "Boolean flag indicating whether to check energy conservation [`false` (default), `true`]"
         arg_type = Bool
-        default = true
-        "--unique_seed"
-        help = "Boolean flag indicating whether to set the random number seed to a unique value [`false` (default), `true`]"
+        default = false
+        "--conservation_softfail"
+        help = "Boolean flag indicating whether to soft fail on conservation errors [`false` (default), `true`]"
+        arg_type = Bool
+        default = false
+        # Output information
+        "--coupler_output_dir"
+        help = "Directory to save output files. Note that TempestRemap fails if interactive and paths are too long. [\"experiments/ClimaEarth/output\" (default)]"
+        arg_type = String
+        default = "experiments/ClimaEarth/output"
```

Comment:
This would be a little weird if you ran from `experiments/ClimaEarth/` (it would create another layer of `experiments/ClimaEarth/`), but I think it's fine for the time being